scaler/package.json

```
@@ -5,7 +5,6 @@
   "author": "Google Inc.",
   "dependencies": {
     "@google-cloud/firestore": "^4.14.1",
```

```suggestion
    "@google-cloud/firestore": "^4.14.1",
    "@google-cloud/functions-framework": "^2.0.0",
```
Instead of removing the functions-framework, just update its version to 2.0.0